package.json

```diff
@@ -22,8 +22,12 @@
     "docsify": "bin/docsify"
   },
   "scripts": {
-  	"test": "ava e2e/index.js",
-    "release": "standard-version"
+    "test": "ava e2e/index.js",
+    "release": "standard-version",
+    "prebuild": "rimraf bin",
+    "build": "rollup -c",
+    "prerelease": "npm run build",
+    "postinstall": "npm run build"
```

Comment:
`postinstall` is not required. Cause we are going to ship the build binary

Reply:
add build step in travis

Reply:
https://travis-ci.org/docsifyjs/docsify-cli/jobs/628384400?utm_medium=notification&utm_source=github_status


`root/e2e/index.js` says:

```js
'use strict';

const path = require('path');
const execa = require('execa');
const test = require('ava');

const rootCommand = path.join(process.cwd(), 'bin/docsify'); // <- does not exists
// ... rest
```

Actually it's  need since `bin/docsify` is versioned anymore. What I can do is add a `pretest` script.

Reply:
it will work , you need to add the build steps in the travis config. like I commented here https://github.com/docsifyjs/docsify-cli/pull/83#discussion_r360712310